### PR TITLE
[1.26] releng: Update release branch jobs for 1.26 

### PR DESCRIPTION
**Pull request by cici37**
releng: Update release branch jobs for 1.26
1.26 rc.0 cut issue:https://github.com/kubernetes/sig-release/issues/2087
Slack thread: https://kubernetes.slack.com/archives/CJH2GBF7Y/p1668515860735989

/sig release
/area release-eng
/assign @jeremyrickard @puerco
cc: https://github.com/orgs/kubernetes/teams/release-engineering

### CODE REVIEW

**Comment by cici37** [comment]
Sorry for missing this file from editing the 1.22 job earlier. Add it back to fix the testgrid dashboard

**Comment by leonardpahlke** [comment]
@kubernetes/sig-release-leads would be good to get reviewed and merged very soon; so ci signal can monitor the 1.26-xy dashboards properly

File: config/jobs/kubernetes/sig-release/release-branch-jobs/1.26.yaml [hunk 1]
```diff
@@ -0,0 +1,1922 @@
...
+    containers:
+    - command:
+      - wrapper.sh
+      - /krel
+      - ci-build
+      - --configure-docker
+      - --allow-dup
+      - --bucket=k8s-release-dev
+      - --registry=gcr.io/k8s-staging-ci-images
+      - --extra-version-markers=k8s-master
```
[the remark this answers is not in the document]
Comment (jeremyrickard) [reply config/jobs/kubernetes/sig-release/release-branch-jobs/1.26.yaml:158]: [Per slack](https://kubernetes.slack.com/archives/CJH2GBF7Y/p1669065861073859),  can we drop the `extra-version-makers` @xmudrii. What is the best course of action to resolve this so the boards are fully populated for the CI signal team to look at asap?

[the remark this answers is not in the document]
Comment (cici37) [reply config/jobs/kubernetes/sig-release/release-branch-jobs/1.26.yaml]: I have update to use `latest-1.26` here. Please let me know if other changes are preferred :)

**Review by justaugustus — APPROVED** [review]
These still need tweaking and I'll try to debug in a follow-up while the boards repopulate.
/lgtm
/approve

File: config/jobs/kubernetes/sig-release/release-branch-jobs/1.26.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (justaugustus) [reply config/jobs/kubernetes/sig-release/release-branch-jobs/1.26.yaml]: Let's continue the discussion on https://github.com/kubernetes/test-infra/pull/28079 and https://github.com/kubernetes/sig-release/issues/850.

**Comment by aojea** [comment]
Folks since this was added it created a new dashboard with jobs that never worked https://github.com/kubernetes/test-infra/pull/28049 , it seems fails to pull some image

can you please take a look on why it fails ?

cc: @dims @Nivedita-coder

**Comment by xmudrii** [comment]
@aojea This issue should be now fixed by #28083 but we'll keep monitoring it.

**Pull request merged**